.github/workflows/unit-tests.yml

```diff
@@ -15,17 +15,11 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: "10"
-      - name: install yarn
-        run: |
-          curl -o- -L https://yarnpkg.com/install.sh | bash
-          echo $HOME/.yarn/bin >>"$GITHUB_PATH"
-          echo $HOME/.config/yarn/global/node_modules/.bin >>"$GITHUB_PATH"
+          cache: yarn
+          node-version: '16'
```

Comment:
need to add it to `.nvmrc`

Reply:
Also, need to change it in the main `jaeger` repo:
```
# ~/dev/jaegertracing/jaeger
$ grep -rn node .github
.github/workflows/ci-release.yml:25:    - uses: actions/setup-node@v3
.github/workflows/ci-release.yml:27:        node-version: '10'
.github/workflows/ci-docker-build.yml:27:    - uses: actions/setup-node@v3
.github/workflows/ci-docker-build.yml:29:        node-version: '10'
.github/workflows/ci-all-in-one-build.yml:26:    - uses: actions/setup-node@v3
.github/workflows/ci-all-in-one-build.yml:28:        node-version: '10'
```

Reply:
I created this this PR : https://github.com/jaegertracing/jaeger/pull/3957